README.md

```diff
@@ -97,7 +99,7 @@
 
 ## Animate
 
-### General
+### General1
```

Comment:
No need to add numbers to the heading, they'll be in different subheadings anyway.
```suggestion
### General
```